views/frontpage.handlebars

```diff
@@ -1,16 +1,30 @@
 <div class="image">
-  <img src="/images/BarCard-banner.jpg" alt="Bar Background" />
+  <img
+    src="/images/BarCard-banner.jpg"
+    class="frontpage-image img-fluid"
+    alt="Bar Background"
+  />
+
+</div>
+<div class="cover-container text-center">
+  <h1 class="text-white mt-4">Welcome to BarCard.</h1>
```

Comment:
Anywhere you have "text-white" should not be needed because we have that on main in main.handlebars. If you need something different like the "text-white-50" then that is fine to stay.